src/utils/auth.ts

```diff
@@ -0,0 +1,22 @@
+import express from 'express';
+const jwt = require('jsonwebtoken');
+
+// TODO: Test out this middle where once login endpoint is done and returns a JSON web token
+const authenticateJWT = (req: any, res: any, next: () => void) => {
+    const authHeader = req.headers.authorization;
+
+    if (authHeader) {
+        const token = authHeader.split(' ')[1];
+        try {
+            jwt.verify(token, process.env.ACCESS_TOKEN_SECRET)
+            .then( () => {
+                next();
+            })
```

Comment:
recommend using await instead of .then, to be consistent (also to make sure the error is caught in your try/catch - with .then, it won't be)